dev.geopjr.Tuba.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: dev.geopjr.Tuba
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.vala
```

Comment:
vala is in the gnome sdk you don't need that. it's not meant for that.

Reply:
Fixed, thanks!